nix/modules/pre-commit.nix

```diff
@@ -0,0 +1,7 @@
+{ self, lib, ... }: {
+  perSystem = { config, self', inputs', pkgs, ... }:
+    {
+      pre-commit.check.enable = true;
```

Comment:
what kind of check is enabled here? is it just the one in the next line?

Reply:
This makes the pre-commit checks become part of `nix flake check`, see: https://flake.parts/options/pre-commit-hooks-nix.html#opt-perSystem.pre-commit.check.enable